src/state_region_config.ts

```diff
@@ -1,67 +1,34 @@
 import { difference, uniq } from 'lodash';
 import { getStateConstants } from './state_constants';
+import { PromisifiedRedisClient } from './redis_client';
+import * as RedisApiUtil from './redis_api_util';
 
-export const stateToRegionMap: { [stateCode: string]: string } = {
-  Alabama: 'Central',
-  Alaska: 'Pacific',
-  Arizona: 'Arizona',
-  Arkansas: 'Central',
-  California: 'California',
-  Colorado: 'Colorado',
-  Connecticut: 'Eastern North',
-  Delaware: 'Eastern North',
-  'District of Columbia': 'Eastern South',
-  Florida: 'Florida',
-  Georgia: 'Georgia',
-  Hawaii: 'Pacific',
-  Idaho: 'Mountain',
-  Illinois: 'Illinois',
-  Indiana: 'Eastern North',
-  Iowa: 'Central',
-  Kansas: 'Central',
-  Kentucky: 'Eastern South',
-  Louisiana: 'Central',
-  Maine: 'Eastern North',
-  Maryland: 'Eastern South',
-  Massachusetts: 'Eastern North',
-  Michigan: 'Eastern North',
-  Minnesota: 'Minnesota',
-  Mississippi: 'Central',
-  Missouri: 'Central',
-  Montana: 'Mountain',
-  Nebraska: 'Central',
-  Nevada: 'Pacific',
-  'New Hampshire': 'Eastern North',
-  'New Jersey': 'Eastern North',
-  'New Mexico': 'Mountain',
-  'New York': 'New York',
-  'North Carolina': 'North Carolina',
-  'North Dakota': 'Central',
-  Ohio: 'Eastern North',
-  Oklahoma: 'Central',
-  Oregon: 'Pacific',
-  Pennsylvania: 'Pennsylvania',
-  'Rhode Island': 'Eastern North',
-  'South Carolina': 'Eastern South',
-  'South Dakota': 'Central',
-  Tennessee: 'Eastern South',
-  Texas: 'Texas',
-  Utah: 'Mountain',
-  Vermont: 'Eastern North',
-  Virginia: 'Eastern South',
-  Washington: 'Pacific',
-  'West Virginia': 'Eastern South',
-  Wisconsin: 'Wisconsin',
-  Wyoming: 'Mountain',
-  National: 'National',
+type StateRegionConfig = {
+  [stateName: string]: string; // mapping of U.S. state name to entrypoint name (region/state name)
 };
 
-export const regionsList =
-  process.env.CLIENT_ORGANIZATION === 'VOTE_AMERICA'
-    ? uniq(Object.values(stateToRegionMap))
+export async function fetchStateRegionConfig(
+  redisClient: PromisifiedRedisClient
+): Promise<StateRegionConfig> {
+  const stateRegionConfig = await RedisApiUtil.getHash(
+    redisClient,
+    'stateRegionConfig'
+  );
+  if (stateRegionConfig) return stateRegionConfig;
+  return {} as StateRegionConfig;
+}
+
+export async function regionsList(
+  redisClient: PromisifiedRedisClient
+): Promise<string[]> {
+  return process.env.CLIENT_ORGANIZATION === 'VOTE_AMERICA'
+    ? uniq(Object.values(fetchStateRegionConfig(redisClient)))
```

Comment:
We can probably remove the client org check if it's in Redis anyway?

Reply:
The other client uses a separate Redis. Do you know if I would need to update their Redis as well for this to work properly?

Reply:
It should? I assume it's an empty hash on the other client, so this function will just return an empty list.

Reply:
Good point. I'll test that out and follow up.

Reply:
It works! Thanks for suggesting that. I pushed a commit with the change, as well as another small but significant change to how fallbacks work if Redis is lacking needed information. Please take a look